Assets/Stack mini game/Scripts/CubeSpawner.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CubeSpawner : MonoBehaviour
+{
+    [SerializeField] private GameObject cubePrefab;
```

Comment:
I am not sure what you mean here. Can I rely move the start position to a scriptable object? If I do so it can't reference something in the scene, as it is an asset. And I need to a gameobject in the scene to get the start position.